source/slang/slang-ir-inst-defs.h

```diff
@@ -432,6 +439,9 @@ INST(SwizzledStore, swizzledStore, 2, 0)
         INST(ifElse, ifElse, 4, 0)
     INST_RANGE(ConditionalBranch, conditionalBranch, ifElse)
 
+    INST(Throw, throw, 1, 0)
+    // tryCall <successBlock> <failBlock> <callee> <err> <args>...
+    INST(TryCall, tryCall, 4, 0)
```

Comment:
I don't follow why this instruction takes an `<err>` operand. Shouldn't the error/value output be expressed as a block parameter of `successBlock` or `failBlock`, respectively?